demonstrations/tutorial_mol_geo_opt.py

```diff
@@ -0,0 +1,427 @@
+r"""
+Optimization of molecular geometries
+====================================
+
+.. meta::
+    :property="og:description": Find the equilibrium geometry of a molecule
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_spectra_h2.png
+
+.. related::
+   tutorial_quantum_chemistry Quantum Chemistry with PennyLane
+   tutorial_vqe Variational Quantum Eigensolver
+   tutorial_givens_rotations Givens rotations for quantum chemistry
+   
+*Author: PennyLane dev team. Last updated: 25 May 2021.*
+
+Predicting the most stable arrangement of the atoms in a molecule is one of the most
+important tasks in quantum chemistry to simulate most properties of a 
```

Comment:
```suggestion
Predicting the most stable arrangement of the atoms in a molecule is an
important task in quantum chemistry, and is needed to simulate many properties of a 
```
Feel free to rephrase if this doesn't convey the intended meaning. Just tried to reduce instances of "most".

Reply:
Thanks @glassnotes